src/stacks-hierarchy/navigation-types.ts

```diff
@@ -86,7 +86,8 @@ type Root_PurchaseAsAnonymousConsequencesScreenParams = {
 type Root_PurchasePaymentScreenParams = {
   offers: ReserveOffer[];
   preassignedFareProduct: PreassignedFareProduct;
-  paymentMethod: PaymentMethod;
+  paymentMethod: PaymentOption;
+  savePaymentMethod: boolean;
```

Comment:
Can `savePaymentMethod` be prefixed with `should` to distinguish it from a function that saves it?

Reply:
Sure, updated now 👍